ROMFS/px4fmu_common/init.d/airframes/4250_teal

```diff
@@ -18,13 +18,8 @@
 # @maintainer Jacob Dahl <jacob.dahl@tealdrones.com>
```

Comment:
@dakejahl should this maintainer be updated?

Reply:
Yeah change it to none

Reply:
Done.  Thanks!

Reply:
Add me as the maintainer for the teal stuff. matt.mcfadden@tealdrones.com. Or let me know how I do that.

@mcsauder thanks for reaching out.

Reply:
@themcfadden you can open a PR changing it here.
https://github.com/PX4/Firmware/blob/1dd6b6a5058199ea6bbf1e2321010abe88a656ee/ROMFS/px4fmu_common/init.d/airframes/4250_teal#L18

Thanks